ros2_ws/src/vision/path_detection/path_detection/obstacles.py

```diff
@@ -107,110 +105,95 @@ def check_range(self, scan, min, max, max_distance):
     # first portion nullifies all data behind the scanner after adjusting min and max to be 0
     # second portion adds potholes based on image data
     # third portion replaces obstacle in front and time of flight sensors
+
+    # this funciton does the following:
+    # 1.
+    # 2.
+    # 3. 
     def lidar_callback(self, scan):
-        # adjust range to only include data in front of scanner\
 
-        # self.get_logger().info(f"trim range: {scan.angle_max}, scan range: {scan.angle_min}")
-        scan.angle_max += math.pi
-        scan.angle_min += math.pi
+        self.get_logger().info(f"angle min: {scan.angle_min} \n angle max: {scan.angle_max} \n angle increment: {scan.angle_increment}")
+
+        scan.angle_max += math.pi                                                   # mirror the incoming scan angles over the y (forward) axis . . .
+        scan.angle_min += math.pi                                                   # flip angle 0 to be on the right and -pi to pi to be on the left
```

Comment:
the comments on these lines don't make a lot of sense. update them so they're clearer
```suggestion
        # change the range from -pi -> pi to 0 -> 2pi for array indexing
        scan.angle_max += math.pi 
        scan.angle_min += math.pi
```